cmd/podman/formats/formats.go

```diff
@@ -125,6 +125,7 @@ func (t StdoutTemplateArray) Out() error {
 			fmt.Fprintln(w, "")
 		}
```

Comment:
Confirmed: removing lines 123-126 yields correct results when output is and is not a tty